app/metrics/jupyterkernelbusyness.go

```diff
@@ -0,0 +1,128 @@
+package metrics
```

Comment:
Since we are heavily refactoring this, let's move to the Go conventions:
1. Change dir structure:
```
cmd/
  sidecar-proxy/
    main.go
pkg/
  sidecar-proxy/
    metricshandler/
      abstract/
        metricshandler.go (package name: abstract)
      jupyterkernelbusyness/
        metricshandler.go (package name: jupyterkernelbusyness)
      numofrequests/
        metricshandler.go (package name: numofrequests)
      metricshandler.go    
    server.go     
```
2. Use `go modules` for dependencies and remove vendor dir
3. Use nuclio logger and the logrus implementation and nuclio errors